caesars-palace/blocks/featured-card/featured-card.css

```diff
@@ -3,6 +3,11 @@
   padding-bottom: var(--layout-size-xl);
 }
 
+.featured-card-wrapper-no-background {
+  padding-bottom: var(--layout-size-xs) !important;
```

Comment:
Should we use !important here?
Is there any other way to handle it?

Reply:
I didn't find one. THing is I need to override the `default` styles to remove the padding and set something else.